integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -390,11 +390,19 @@ case class LoadTable(
       val useKettle = options.get("use_kettle") match {
         case Some(value) => value.toBoolean
         case _ =>
-          val useKettleLocal = System.getProperty("use.kettle")
+          var useKettleLocal = System.getProperty("use.kettle")
+          if (useKettleLocal == null && sqlContext.sparkContext.getConf.contains("use_kettle")) {
```

Comment:
Same comments as jackly.  i suggest using  "useKettle", what is your opinion? @jackylk